values.yaml

```diff
@@ -8,9 +8,12 @@ nodes:
     user: 'node-1'
     password: 'p@$swr0d44'
     shared_path: '/tmp/org-1/servermedias'
+    # minikube: True
```

Comment:
Maybe you could move the minikube option to the `options` section? Since the value is currently only checked for the first node, it behaves like a general setup option and not something that may change on a node-to-node basis.

Reply:
Yes, you're right !